modules/core/test/v2/unit/coins/trx.ts

```diff
@@ -20,6 +43,72 @@ describe('TRON:', function() {
     basecoin.should.be.an.instanceof(Trx);
   });
 
+  it('explain a txHex', co(function *() {
+    const explainParams = {
+      txHex: mockTx.raw_data_hex,
+      fee: 1,
+      txID: mockTx.txID,
+    };
+    const explanation = yield basecoin.explainTransaction(explainParams);
+    const toAddress = Tron.getBase58AddressFromHex(mockTx.raw_data.contract[0].parameter.value.to_address);
+    explanation.id.should.equal(mockTx.txID);
+    explanation.outputs.amount.should.equal(10);
+    explanation.outputAmount.should.equal(10);
+    explanation.changeAmount.should.equal(0);
+    explanation.changeOutputs.length.should.equal(0);
+    explanation.fee.should.equal(1);
+    explanation.expiration.should.equal(mockTx.raw_data.expiration);
+    explanation.timestamp.should.equal((mockTx.raw_data.timestamp));
+    explanation.outputs.address.should.equal(toAddress);
+  }));
+
+  it('should throw if txHex does not have the proper format', co(function *() {
+    const explainParams = {
+      fee: 1,
+      txID: mockTx.txID,
+      txHex: 'this_is_an_odd_length_string'
+    };
+    try {
+      yield basecoin.explainTransaction(explainParams);
```

Comment:
Could replace whole try/catch with `yield basecoin.explainTransaction(explainParams).should.be.rejectedWith('missing valid txHex string');`